src/pages/index.html

```diff
@@ -1,68 +1,68 @@
-                    <div class="titlebar-button" id="file">File</div>
-                    <div class="titlebar-button" id="view">View</div>
-                </div>
-            </div>
-            <div id="titlebar-content-right">
-                <div id="logo">
-                    <img src="../components/images/scplogo.png" alt="" width="35px">
-                    <div id="logo-text">
-                        <div>SCP FOUNDATION</div>
-                        <div>SECURE. CONTAIN. PROTECT.</div>
-                    </div>
-                </div>
-                <div id="titlebar-buttons">
-                    <div class="titlebar-button" id="minimize"></div>
-                    <div class="titlebar-button" id="maximize"></div>
-                    <div class="titlebar-button" id="close"></div>
-                </div>
-            </div>
+  <body>
+    <div id="titlebar">
+      <div id="titlebar-content-left">
+        <div id="menu-buttons">
+          <div class="titlebar-button" id="file">File</div>
+          <div class="titlebar-button" id="view">View</div>
         </div>
-
-        <input type="checkbox" id="openmenu" class="openmenu">
-        <div id="sidebar">
-            <span>SCP History</span>
-            <hr>
-            <div id="scp-list">
-                <ul></ul>
-            </div>
+      </div>
+      <div id="titlebar-content-right">
+        <div id="logo">
+          <img src="../components/images/scplogo.png" alt="" width="35px" />
+          <div id="logo-text">
+            <div>SCP FOUNDATION</div>
+            <div>SECURE. CONTAIN. PROTECT.</div>
+          </div>
         </div>
-        <div id="sidetoolbar">
-            <label for="openmenu" id="sidebarbutton">H</label>
+        <div id="titlebar-buttons">
+          <div class="titlebar-button" id="minimize"></div>
+          <div class="titlebar-button" id="maximize"></div>
+          <div class="titlebar-button" id="close"></div>
         </div>
+      </div>
+    </div>
 
-        <div id="toolbar">
-            <div id="personnel-text">
-                <div class="personnel-title">SCP Containment Specialist (Level 5)</div>
-                <div class="personnel-site">SITE-19</div>
-            </div>
-        </div>
-        <div class="term-container">
-            <div class="terminal">
-                <div class="window cursor-block" id="window"></div>
-            </div>
-        </div>
-    </body>
-    <script src="../components/scripts/terminal.js"></script>
-    <script src="../renderer.js"></script>
+    <input type="checkbox" id="openmenu" class="openmenu" />
+    <div id="sidebar">
+      <span>SCP History</span>
+      <hr />
+      <div id="scp-list">
+        <ul></ul>
+      </div>
+    </div>
+    <div id="sidetoolbar">
+      <label for="openmenu" id="sidebarbutton">H</label>
+    </div>
+
+    <div id="toolbar">
+      <div id="personnel-text">
+        <div class="personnel-title">SCP Containment Specialist (Level 2)</div>
```

Comment:
Containment specialists usually have level 2 clearance (see [the SCP Wiki](https://scp-wiki.wikidot.com/security-clearance-levels#toc15))

Reply:
I have it as 5 because a level 2 assumes restricted access to files. Plus the title is just a placeholder for now. In the future I might have a feature to customize it.